ide-support/revsaveasstandalone.livecodescript

```diff
@@ -312,6 +312,9 @@ command revDoSaveAsStandalone pStack, pFolder, pSettings, @xStandalonePaths
       put the result into tOutputFolder
       if there is no folder tOutputFolder then
          create folder tOutputFolder
+         if the result is not empty then
```

Comment:
@livecodepanos I suspect that this is what's causing the test failures but I'm not sure why yet

Reply:
yes `Cannot create a folder '/Users/travis/build/livecode/livecode/ide/_tests/_build/Stack 1536671068.2'` etc

So I guess the next question is why not!

Reply:
OK in the standalone building test `_support.livecodescript` there is `GetTargetFolder` which returns the ide repo & `_tests/_build` but does not ensure that that folder exists. I think if you change this to use `revSBEnsureFolder` it will work and actually be an improvement.

Reply:
Thanks @montegoulding I will have a look and update the PR